src/current/v25.1/monitor-and-analyze-transaction-contention.md

```diff
@@ -0,0 +1,586 @@
+
+#### Transactions page
+
+The Transactions page ([CockroachDB Cloud Console]({% link cockroachcloud/transactions-page.md %}) or [DB Console]({% link {{ page.version.version }}/ui-transactions-page.md %})) shows transactions that were involved in contention events. To view these transactions, change the search criteria to view the top 100 transaction fingerprints by Contention Time, then sort descending by Contention Time column.
+
+The Contention Time column displays the average and standard deviation of the time spent in contention. As noted earlier, only lock contention events are included in the contention time.
+
+The following image shows the Transactions page with the top 3 transactions fingerprints by Contention Time in a cluster containing the test data in the [Analyze using `crdb_internal` tables](#analyze-using-crdb_internal-tables) section.
+
+<img src="{{ 'images/v25.1/contention-3.png' | relative_url }}" alt="Transactions page by Contention Time" style="border:1px solid #eee;max-width:100%" />
+
+### Insights page
+
+The Insights page ([CockroachDB Cloud Console]({% link cockroachcloud/insights-page.md %}) or [DB Console]({% link {{ page.version.version }}/ui-insights-page.md %})) displays a variety of [workload insights]({% link {{ page.version.version }}/ui-insights-page.md %}#workload-insights-tab) on statement and transaction executions related to transaction contention:
+
+- [Slow Execution]({% link {{ page.version.version }}/ui-insights-page.md %}#slow-execution)
+
+    A statement or transaction is classified as Slow Execution if it meets either of the following criteria:
+    
+    - The execution took longer than the configured [`sql.insights.latency_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-insights-latency-threshold).
+    - Anomaly detection is enabled ([`sql.insights.anomaly_detection.enabled`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-insights-anomaly-detection-enabled)) and the execution was greater than twice the median execution latency and higher than [`sql.insights.anomaly_detection.latency_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-insights-anomaly-detection-latency-threshold).
+
+- [High Contention]({% link {{ page.version.version }}/ui-insights-page.md %}#high-contention)
+
+    After meeting the Slow Execution criteria, an execution is classified as High Contention if a significant portion of its duration was due to contention.
+
+- [Failed Execution]({% link {{ page.version.version }}/ui-insights-page.md %}#slow-execution)
+
+    A statement or transaction that failed due to a serialization error is identified as a Failed Execution. If the serialization error was due to contention, there is a conflicting or blocking transaction recorded. The execution details will show the conflicting transaction.
```

Comment:
We capture all types of failures but for serialization errors, we capture more info (conflicting info) to help troubleshoot.